gatsby-node.js

```diff
@@ -77,9 +77,10 @@ exports.onCreateNode = async ({
   createNodeId,
 }) => {
   const { createNodeField, createNode } = actions
-  if (node.internal.type === `apps`) {
+  if (node.internal.type === `apps` && !!node.imageUrl.trim()) {
```

Comment:
This prevents to add the source code lastCommit to the nodes. Should be in a separate if clause, shouldn't it?

Reply:
yes sure I can change that